CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### Changed
 - add docker compose deployment
 - update default microk8s to 1.30
+- change group and user ID in Docker image to 10001:10001
+- create and/or change ownership of /tmp and /.pysnmp to 10001:10001 in Dockerfile
+- while mounting volumes in docker compose explicitly mount the as read only or read write
```

Comment:
typo with `the` I think there should be `them`

Reply:
done